### PR TITLE
Gamma x

### PR DESCRIPTION
**Pull request by empEvil**
This pull request ensures we save the GammaX correlations from TreeCorr.

- It saves the GammaX to a separate sacc file.
- It uses the shot noise from TreeCorr to calculate the error and the covariance.
- When plotting with the stage TwoPointPlots, it now also plots the GammaX.

### CODE REVIEW

File: examples/laptop_pipeline.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ stages:
     - name: TXPSFDiagnostics     # Compute and plots other PSF diagnostics
     - name: TXBrighterFatterPlot # Make plots tracking the brighter-fatter effect
     - name: TXPhotozPlots        # Plot the bin n(z)
+    #- name: TXTwoPGXP            # Plot the gammaX correlation
```
Comment (jpratmarti) [inline examples/laptop_pipeline.yml:35]: Is this supposed to be commented or should we comment it out?

Comment (empEvil) [reply examples/laptop_pipeline.yml]: Was a left over from trying to get plotting to work

**Review by jpratmarti — APPROVED** [review]

File: examples/laptop_pipeline.yml (hunk 1 above)
Comment (jpratmarti) [reply examples/laptop_pipeline.yml]: Actually @empEvil I just seen that you have some conflicts with master, so you should probably resolve this before merging (sorry I approved too fast! :) )

**Comment by empEvil** [comment]
I think I fixed the last outstanding conflicts with master

**Review by jpratmarti — APPROVED** [review]

**Review by joezuntz — CHANGES_REQUESTED** [review]
This looks great - I've requested one very minor change.

**Pull request merged**